README.md

```diff
@@ -16,6 +16,13 @@ There are a few open-source projects that can convert Java objects to JSON. Howe
 
 ### Download
 
+Gradle.kts:
```

Comment:
I would put this after the existing Gradle one since I think Groovy-Gradle is still commoner.